framework-crates/objc2-virtualization/translation-config.toml

```diff
@@ -0,0 +1,16 @@
+framework = "Virtualization"
+crate = "objc2-virtualization"
+required-dependencies = ["objc2-app-kit", "objc2-foundation"]
+macos = "12.0"
```

Comment:
> Since my VMM is designed to run macOS guests, the `objc2-virtualization` must support the APIs for macOS 12.0 or later. So I have hardcoded the minimum supported macOS version to 12.0 in `framework-crates/objc2-virtualization/translation-config.toml`. This may not be appropriate for other use cases.

You should specify the version that it says on the front page, i.e. macOS 11.0. Newer APIs will still be available, the specified version in `translation-config.toml` is mostly for CI.